jib-core/src/test/java/com/google/cloud/tools/jib/image/json/ImageToJsonTranslatorTest.java

```diff
@@ -136,6 +141,26 @@ public void testPortListToMap() {
     Assert.assertEquals(expected, ImageToJsonTranslator.portListToMap(input));
   }
 
+  @Test
+  public void testVolumeListToMap() {
+    ImmutableList<AbsoluteUnixPath> input =
+        ImmutableList.of(
+            AbsoluteUnixPath.get("/var/job-result-data"),
+            AbsoluteUnixPath.get("/var/log/my-app-logs"));
+    ImmutableSortedMap<String, Map<?, ?>> expected =
+        ImmutableSortedMap.of(
+            "/var/job-result-data", ImmutableMap.of(), "/var/log/my-app-logs", ImmutableMap.of());
+    Assert.assertEquals(expected, ImageToJsonTranslator.volumesListToMap(input));
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void testInvalidVolumeListToMap() {
+    ImmutableList<AbsoluteUnixPath> input =
+        ImmutableList.of(AbsoluteUnixPath.get("'/var/job-result-data"));
+
+    ImageToJsonTranslator.volumesListToMap(input);
```

Comment:
So for example, here, we actually have a line that would never be reached since the test actually throws the `IllegalArgumentException` on line 159.

Reply:
Actually this test doesn't even make sense. There is no way to get an invalid `AbsoluteUnixPath` instance. Don't know what I tried to do here, removing this method.